### PR TITLE
Home Task 2

### PR DESCRIPTION
**Pull request by tetyana2512**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,50 +1,119 @@
...
+// // Task 4
+// let a = 1, b = 2;
+// result = (a + b >= 3) ? 'Yep' : 'Noup';
+//
+// // Task 5
+// let name = (name = 'admin') ? 'Hi':
+//     (name = 'manager') ? 'Hello':
+//         (name = '') ? 'No login':
+//             '';
+// Home Work 2
```
Comment (dmitry-darly-solutions) [inline script.js]: move Home Work 2  to separate file

Comment (dmitry-darly-solutions) [reply script.js]: add .idea to .gitignore and remove folder from repository

File: script.js [hunk 2]
```diff
@@ -1,50 +1,119 @@
...
-if (login === 'admin'){
-    let pass = prompt('Введите пароль')
-    if (pass === 'passw0rd'){
-        alert('Welcome home!');
+let simpleNum = [ ];
+for (let n = 2; n <= 10; n++){
+    let isSimple = true;
+    for (let m = 2 ; m <= n-1 ; m++){
+       isSimple = isSimple ? !!(n % m) : isSimple;
+        // console.log('for m', m , n , n % m , isSimple)
```
Comment (dmitry-darly-solutions) [inline script.js]: remove console.log

File: script.js [hunk 3]
```diff
@@ -1,50 +1,119 @@
...
     }
-    else if (pass === null){
-        alert('Canceled')
+    if (isSimple){
+        simpleNum.push (n);
+    }
+}        console.log(simpleNum)
```
Comment (dmitry-darly-solutions) [inline script.js]: remove console.log

File: script.js [hunk 4]
```diff
@@ -1,50 +1,119 @@
...
+
+// Task 3
+for (let n = 1; n <=100 ; n++){
+    if ((n % 3) && (n % 5)){
+        console.log(n)
+    } else if (!(n % 3)){
+        console.log('fith');
```
Comment (dmitry-darly-solutions) [inline script.js]: ‘Fizz’

File: script.js [hunk 5]
```diff
@@ -1,50 +1,119 @@
...
+    } else {
+        console.log('buth');
```
Comment (dmitry-darly-solutions) [inline script.js]: Buzz

File: script.js [hunk 6]
```diff
@@ -1,50 +1,119 @@
...
-    else {
-        alert('Wrong password');
-        }
 }
-else if (login === null){
-    alert('Canceled');
+// Task 4
+for (let n = 1; n <=100 ; n++){
+    if (!(n % 3) && !(n % 5)) {
+        console.log(n)
```
Comment (dmitry-darly-solutions) [inline script.js]: FizzBuzz

File: script.js [hunk 7]
```diff
@@ -1,50 +1,119 @@
...
+    if ((n % 2)) {
+        string += '#'
+    } else {
+        string += ' ';
+    }
+    if (!(n % 7)){
+        string += '\n'
+    }
 }
+console.log(string)
```
Comment (dmitry-darly-solutions) [inline script.js]: remove console.log

File: script.js [hunk 8]
```diff
@@ -1,50 +1,119 @@
...
+let base = prompt('Введите число, которое хотите возвести в степень');
+let n = prompt('Введите число степени');
+let result = Math.pow(base, n);
+console.log(result)
+
+
+
+
+
+// Выведите столбец четных чисел в промежутке от 0 до 10.
```
Comment (dmitry-darly-solutions) [inline script.js:112]: remove this part

File: script.js [hunk 9]
```diff
@@ -1,50 +1,119 @@
...
+    if (!(n % 7)){
+        string += '\n'
+    }
 }
+console.log(string)
+// Task 5
+let base = prompt('Введите число, которое хотите возвести в степень');
+let n = prompt('Введите число степени');
+let result = Math.pow(base, n);
+console.log(result)
```
Comment (dmitry-darly-solutions) [inline script.js]: remove console

File: script.js [hunk 10]
```diff
@@ -1,50 +1,119 @@
...
+    }
+    if (!(n % 7)){
+        string += '\n'
+    }
 }
+console.log(string)
+// Task 5
+let base = prompt('Введите число, которое хотите возвести в степень');
+let n = prompt('Введите число степени');
+let result = Math.pow(base, n);
```
Comment (dmitry-darly-solutions) [inline script.js]: please make this via cycle without Math.pow

File: hw2/Home-Work-2.js [hunk 11]
```diff
@@ -0,0 +1,51 @@
...
+    } else if (!(n % 3)){
+        console.log('Fizz');
+    } else {
+        console.log('Buzz');
+    }
+}
+// Task 4
+for (let n = 1; n <=100 ; n++){
+    if (!(n % 3) && !(n % 5)) {
+        console.log(FizzBuzz)
```
Comment (dmitry-darly-solutions) [inline hw2/Home-Work-2.js:32]: FizzBuzz should be a string

**Comment by tetyana2512** [comment]
Update

**Review by dmitry-darly-solutions — APPROVED** [review]

**Pull request merged**